hack/lib/init.sh

```diff
@@ -31,6 +31,10 @@ unset CDPATH
 # they can explicitly set GO111MODULE=on
 export GO111MODULE=off
 
+if [[ "${KUBE_PROVIDERLESS:-"off"}" == "on" ]]; then
```

Comment:
tiny nit: we usually use "y", "n" for our own build options? (like KUBE_COVER=y, KUBE_BUILD_CONFORMANCE = y, ), some places accept `true` or any non-default value but most of the build opts are y/n

Reply:
switching to "y" and "n". good call.